src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -499,6 +499,14 @@ protected virtual IntPtr WndProc(IntPtr hWnd, uint msg, IntPtr wParam, IntPtr lP
                         ScreenToClient(DipFromLParam(lParam)),
                         new Vector(0, ((int)wParam >> 16) / wheelDelta), GetMouseModifiers(wParam));
                     break;
+                case UnmanagedMethods.WindowsMessage.WM_MOUSEHWHEEL:
```

Comment:
Tiny nit! Could we get a blank line below the `break` to follow the style for the rest of the file?